appveyor.yml

```diff
@@ -0,0 +1,31 @@
+# Test against this version of Node.js
+environment:
+  matrix:
+    # node.js
+    - nodejs_version: "0.10"
+    - nodejs_version: "0.12"
+    # io.js
+    - nodejs_version: "1.0"
+
+# Install scripts. (runs after repo cloning)
+install:
+  # Get the latest stable version of Node.js or io.js
+  - ps: Install-Product node $env:nodejs_version
+  # Ensure the latest npm version
+  - npm install -g npm
```

Comment:
At the moment a lot of this stuff is not working :( but you are right, we can probably get rid of this line